sdks/python/apache_beam/pvalue.py

```diff
@@ -666,15 +666,14 @@ def as_dict(self):
     return dict(self.__dict__)
 
   def __iter__(self):
-    for _, value in sorted(self.__dict__.items()):
+    for _, value in self.__dict__.items():
       yield value
 
   def __repr__(self):
-    return 'Row(%s)' % ', '.join(
-        '%s=%r' % kv for kv in sorted(self.__dict__.items()))
+    return 'Row(%s)' % ', '.join('%s=%r' % kv for kv in self.__dict__.items())
 
   def __hash__(self):
-    return hash(type(sorted(self.__dict__.items())))
+    return hash(type(self.__dict__.items()))
```

Comment:
I tried running this interactively but got the same hash:
```
>>> hash(type({}.items()))
5913863196444
>>> hash(type({1:2}.items()))
5913863196444
```

Is it working as intended?

I guess this is technically okay according to the docs, but probably not what you wanted: `The only required property is that objects which compare equal have the same hash value`.
[ref](https://docs.python.org/3/reference/datamodel.html#object.__hash__)

Reply:
I didn't write the original `__hash__` implementation that uses `type()`. You're right that it seems like an odd choice though. It seems likely this was a copy-paste error or a typo.

Reply:
Ok, went ahead and pushed a commit to fix this.